concourse/steps/replicate_secrets.mako

```diff
@@ -99,11 +103,32 @@ replicate_secrets(
 
 logger.info('generating cfg element status report')
 
-status_reports = cmu.generate_cfg_element_status_reports('${cfg_repo_relpath}')
+status_reports = cmu.generate_cfg_element_status_reports(
+  cfg_dir='${cfg_repo_relpath}',
+  element_storage='${cfg_repo_url}',
+)
 cmr.create_report(status_reports)
 cfg_report_summary_gen = cmr.cfg_element_statuses_storage_summaries(status_reports)
 cfg_responsible_summary_gen = cmr.cfg_element_statuses_responsible_summaries(status_reports)
 
+% if compliance_reporting_repo_url:
+## use default from `CONCOURSE_CURRENT_CFG` once it refers to proper cfg_set
```

Comment:
let us rather keep passing-in repository explicitly at _this_ point. -> no need for TODO/comment.

Reply:
comment refers to delivery svc

Reply:
mved it some lines